verticaltabs.jsm

```diff
@@ -589,16 +590,20 @@ VerticalTabs.prototype = {
       aTab.setAttribute('crop', 'end');
     }
 
-    aTab.addEventListener('load', function () {
-      let tab_address = aTab.linkedBrowser.currentURI.spec;
-      //use to set preview image as metadata image 2/4
-      // document.getAnonymousElementByAttribute(aTab, 'anonid', 'tab-meta-image').style.backgroundImage = `url(${this.getPageMetaDataImage(aTab)}`;
+    aTab.addEventListener('load', () => {
+      let tab_meta_image = document.getAnonymousElementByAttribute(aTab, 'anonid', 'tab-meta-image');
+      tab_meta_image.style.backgroundImage = `url(moz-page-thumb://thumbnail/?url=${encodeURIComponent(aTab.linkedBrowser.currentURI.spec)}), url(resource://tabcenter/skin/blank.png)`;
 
-      //use to set preview image as screenshot
-      document.getAnonymousElementByAttribute(aTab, 'anonid', 'tab-meta-image').style.backgroundImage = `url(moz-page-thumb://thumbnail/?url=${encodeURIComponent(tab_address)}), url(resource://tabcenter/skin/blank.png)`;
-
-      document.getAnonymousElementByAttribute(aTab, 'anonid', 'address-label').value = tab_address;
-    }.bind(this));
+      if (aTab.getAttribute('VTLastUrl') !== aTab.linkedBrowser.currentURI.spec) {
```

Comment:
So, since `aTab` is a Javascript object (as well as being a DOM element), I think should be able to use `aTab.VTLastUrl`, instead of `getAttribute` and `setAttribute`…